docs/src/components/ActionButtonFooter.astro

```diff
@@ -51,8 +51,6 @@ const { data } = await getEntry('i18n', lang);
   .button-sponsor {
     background-color: red;
     border-color: red;
-    color: white;
+    color: white !important;
```

Comment:
indeed this is working fine for that time, but it's often better to use CSS variables because some colors differ from dark and light themes.